test/Operators/hybrid/2d.jl

```diff
@@ -172,63 +188,154 @@ end
+            Ic2f = Operators.InterpolateC2F(
+                top = Operators.Extrapolate(),
+                bottom = Operators.Extrapolate(),
+            )
+            divf2c = Operators.DivergenceF2C()
+            # only upward component of divergence
+            @. dh = divf2c(Ic2f(h) * Geometry.WVector(1.0))
 
-    # A horizontal Gaussian blob, centered at (-x_0, -z_0),
-    # with `a` the height of the blob and σ the standard deviation
-    function h_blob(x_0, z_0, a = 1.0, σ = 0.2)
-        coords = Fields.coordinate_field(hv_center_space)
-        f = map(coords) do coord
-            a * exp(-((coord.x + x_0)^2 + (coord.z + z_0)^2) / (2 * σ^2))
+            # only horizontal component of divergence
+            hdiv = Operators.Divergence()
+            @. dh += hdiv(h * Geometry.UVector(1.0)) # add the two components for full divergence +=
+            Spaces.weighted_dss!(dh)
+
+            return divuf
         end
 
-        return f
-    end
+        # A horizontal Gaussian blob, centered at (-x_0, -z_0),
+        # with `a` the height of the blob and σ the standard deviation
+        function h_blob(x_0, z_0, a = 1.0, σ = 0.2)
+            coords = Fields.coordinate_field(hv_center_space)
+            f = map(coords) do coord
+                a * exp(-((coord.x + x_0)^2 + (coord.z + z_0)^2) / (2 * σ^2))
+            end
 
-    # Spatial derivative of a horizontal Gaussian blob, centered at (-x_0, -z_0),
-    # with `a` the height of the blob and σ the standard deviation
-    function div_h_blob(x_0, z_0, a = 1.0, σ = 0.2)
-        coords = Fields.coordinate_field(hv_center_space)
-        div_uf = map(coords) do coord
-            -a * (exp(-((coord.x + x_0)^2 + (coord.z + z_0)^2) / (2 * σ^2))) /
-            (σ^2) * ((coord.x + x_0) + (coord.z + z_0))
+            return f
         end
 
-        return div_uf
-    end
+        # Spatial derivative of a horizontal Gaussian blob, centered at (-x_0, -z_0),
+        # with `a` the height of the blob and σ the standard deviation
+        function div_h_blob(x_0, z_0, a = 1.0, σ = 0.2)
+            coords = Fields.coordinate_field(hv_center_space)
+            div_uf = map(coords) do coord
+                -a * (exp(-((coord.x + x_0)^2 + (coord.z + z_0)^2) / (2 * σ^2))) / (σ^2) * ((coord.x + x_0) + (coord.z + z_0))
+            end
+
+            return div_uf
+        end
 
-    f = Fields.FieldVector(h = h_blob(0.5, 0.5))
+        f = Fields.FieldVector(h = h_blob(0.5, 0.5))
 
-    divuf = divergence(f)
-    exact_divuf = div_h_blob(0.5, 0.5)
+        divuf = divergence(f)
+        exact_divuf = div_h_blob(0.5, 0.5)
 
-    @test norm(exact_divuf .- divuf.h) ≤ 5e-2
+        err[k] = norm(exact_divuf .- divuf.h)
+    end
+    # Divergence convergence rate
+    conv_comp_div = convergence_rate(err, Δh)
+    @test err[3] ≤ err[2] ≤ err[1] ≤ 0.1
+    @test conv_comp_div[1] ≈ 0.5 atol = 0.1
+    @test conv_comp_div[2] ≈ 0.5 atol = 0.1
+    @test conv_comp_div[3] ≈ 0.5 atol = 0.1
```

Comment:
Why are these still at 0.5 order?

Reply:
Because this test uses the Extruded BC. The one that uses Dirichelet (SetValue) are 2nd order